apps/testing-javascript/src/lib/purchases.ts

```diff
@@ -3,7 +3,7 @@ import {Purchase} from '@skillrecordings/database'
 
 export async function updatePurchaseStatusForCharge(
   chargeId: string,
-  status: 'Valid' | 'Refunded' | 'Disputed' | 'Banned',
+  status: 'Valid' | 'Refunded' | 'Disputed' | 'Banned' | 'Restricted',
```

Comment:
This function is appearing in every app and makes two calls directly against Prisma -- this is a good candidate for de-duping and moving into the `prisma-api.ts`.